cleanup_utils/requirements.txt

```diff
@@ -1,6 +1,7 @@
-boto3==1.9.61
-mysqlclient==1.3.9
-python-novaclient==6.0.0
-PyYAML==3.10
-Django==1.10.7
-
+boto3==1.9.171
+mysqlclient==1.4.2.post1
+python-novaclient==14.1.0
+PyYAML==3.13
+Django==2.2.2
+django-environ==0.4.5
+virtualenv==16.6.1
```

Comment:
Yes. I got CircleCI tasks whose cleanup phase specifically failed because of not having the virtualenv package (and I think the django-environ package too).